.travis.yml

```diff
@@ -66,6 +69,14 @@ matrix:
         - DESC="package no error"
         - CMD="mvn clean package -Passembly"
 
+    # checkstyle regression no exception on latest commit (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="checkstyle regression no exception on latest commit"
+        - CMD1="./.ci/travis/checkstyle_regression_no_exception.sh HEAD HEAD^ &&"
+        - CMD2="./.ci/travis/checkstyle_regression_no_exception.sh 44efd23 4b66933"
```

Comment:
3rd parameter must be release name.
If we are keeping `HEAD` for now, than skip this process if 3rd parameter is `null`.